components/GridToolBox/index.js

```diff
@@ -109,10 +109,10 @@ class GridToolBox extends Component {
 
         if (filterElement.type === filterElementTypes.datePickerBetween || filterElement.type === filterElementTypes.dateTimePickerBetween) {
             filterValue = {
-                from: filters.hasOwnProperty(filterElement.name.from)
+                from: filters.hasOwnProperty(filterElement.name) && filters.hasOwnProperty(filterElement.name.from)
```

Comment:
Did you mean from: filters.hasOwnProperty(filterElement.name && filterElement.name.from)